### PR TITLE
Improve blueprint structure for proper release JAR creation

### PR DESCRIPTION
**Pull request by mikezaccardo**
- [x] Rename catalog files
- [x] Properly reference template JSON files from the classpath
- [x] Test single-cluster deployment
- [x] Test multi-cluster deployment
- [x] Update documentation to cover new installation process

TODO Later:
- Add Travis CI tests
- Secret keys should be a true Java List data type
- Eventually single- and multi-cluster should share more common entities -- right now it is very WET (due to just-significant-enough differences)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+         <resource>
+            <directory>resources</directory>
+            <filtering>false</filtering>
+         </resource>
+      </resources>
+   </build>
+
+   <profiles>
+      <profile>
+         <id>sonatype-release</id>
```
Comment (andreaturli) [inline pom.xml:201]: please document how to use this `profile`

**Review by andreaturli — COMMENTED** [review]
minor comment, haven't tested it though

File: pom.xml (hunk 1 above)
Comment (grkvlt) [reply pom.xml:201]: same as `brooklyncentral/clocker` and so on, it's one of our defaults

**Review by grkvlt — APPROVED** [review]
Looks good @mikezaccardo

**Pull request merged**